contracts/ethereum/OCR2Aggregator.go

```diff
@@ -0,0 +1,3889 @@
+// Code generated - DO NOT EDIT.
```

Comment:
Can you move this to the chainlink repo? Shouldn't be adding even more contracts to the mostly deprecated charts here.